data/mods/CBM_Dungeon_Test/LS_furniture_and_terrain.json

```
@@ -0,0 +1,455 @@
+[
+  {
+    "type": "terrain",
+    "id": "t_nl_mist",
+    "name": "swirling mist",
+    "description": "A swirling, exotic pattern of shapes, floating in an ill-defined darkness.  Staring at it too long hurts your brain.",
+    "symbol": "LINE_OXOX",
+    "color": "green",
+    "coverage": 100,
+    "move_cost": -1,
+    "flags": [ "NOITEM", "SUPPORTS_ROOF", "WALL", "NO_SCENT", "AUTO_WALL_SYMBOL", "BLOCK_WIND" ]
+  },
+  {
+    "type": "terrain",
+    "id": "t_nl_chasm",
+    "name": "sparkling chasm",
+    "description": "At a glance this just looks like darkness.  Then, gazing deeper, you think it might be full of stars.  Still deeper, perhaps those stars are eyes?",
+    "symbol": "*",
+    "color": "dark_gray",
+    "move_cost": -1,
```

```suggestion
    "move_cost": 1,
```
This should also be possible to move onto.